helm/alfresco-content-services/charts/alfresco-search/templates/_helpers-image-pull-secrets.tpl

```diff
@@ -0,0 +1,7 @@
+{{- define "alfresco-content-services.imagePullSecrets" }}
+{{- if .Values.global.alfrescoRegistryPullSecrets }}
+# only set this secret if a private docker registry variable is defined 2
+imagePullSecrets:
+  - name: {{ .Values.global.alfrescoRegistryPullSecrets }}
+{{- end }}
+{{- end }}
```

Comment:
to get rid of this file I think you just need to add in the alfresco-search [Chart.yml](https://github.com/Alfresco/acs-deployment/blob/master/helm/alfresco-content-services/charts/alfresco-search/Chart.yaml) the dependency to alfresco-content-common like has been done in another subchart like [here](https://github.com/Alfresco/acs-deployment/blob/master/helm/alfresco-content-services/charts/alfresco-elasticsearch-connector/Chart.yaml#L8)